### ISSUE TITLE
Orders table column get removed by dukan's filter

### ISSUE THREAD

**Issue by iamdharmesh** [Knawat/dropshipping-woocommerce#16]
Orders table column is not there when dukan plugin is installed.

**Comment by iamdharmesh** [comment]
Created issue at Dokan repo https://github.com/weDevsOfficial/dokan/issues/254

**Comment by HazemKhaled** [comment]
Do we need to set this version as minimum?

**Comment by iamdharmesh** [comment]
Yes, I have thought to fix issue at our end regardless of dokan version but, Dokan is using [`manage_edit-{$post_type}_columns`](https://codex.wordpress.org/Plugin_API/Filter_Reference/manage_edit-post_type_columns) for modify columns which has been supplanted by [`manage_{$post_type}_posts_columns`](https://codex.wordpress.org/Plugin_API/Action_Reference/manage_posts_custom_column). So, I have keep using latest filter for add column at our end and reported issue at Dokan
